storybookjs/storybook
# extend documentation on how to use knob in Angular

Issue:

## What I did
I extended documentation on how to use knob inputs in Angular

## How to test

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
Maybe we could add one?

Does this need an update to the documentation?
N/A

If your answer is yes to any of these, please make sure to include it in your PR.
